cytominer_database/write.py

```diff
@@ -0,0 +1,61 @@
+import csv
+import click
+import warnings
+import zlib
+import pandas as pd
+import backports.tempfile
+import sqlalchemy.exc
+from sqlalchemy import create_engine
+import pyarrow
+import pyarrow.parquet as pq
+import pyarrow.csv
+import numpy as np
+import collections
+import numpy as np
+import cytominer_database
+import cytominer_database.utils
+import cytominer_database.load
+
+
+def write_to_disk(dataframe, table_name, output_path, engine, writers_dict):
```

Comment:
This is tested by the main testing function: If write_to_disk() does not write the correct file to disk, then it cannot be loaded and compared and the test test_ingest_incl_parquet.py would fail. Note that we test both the SQLite and the Parquet case of the function.
Also: It's not possible to create an isolated test for this function (because its input in generated over many previous steps), unless the dictionary is generated manually and the parquet writer is opened manually. I.e. hard-coding all other function outputs, except for this one.